examples/jsm/controls/OrbitControls.js

```diff
@@ -248,6 +250,7 @@ class OrbitControls extends EventDispatcher {
 				offset.applyQuaternion( quatInverse );
 
 				position.copy( scope.target ).add( offset );
+				scope.object.parent ? scope.object.parent.worldToLocal( position ) : scope.object.position.copy( position )
```

Comment:
`scope.object.parent.worldToLocal( position )` does not update `scope.object.position`. Can you try this please:
```js
if ( scope.object.parent !== null ) {

    scope.object.worldToLocal( position );

}

scope.object.position.copy( position );
```

Reply:
Or ideally just this:
```js
scope.object.worldToLocal( position );
scope.object.position.copy( position );
```

Reply:
It does both result in a flickering camera (as above)

``position`` is actually the object.position Vector3 no because of this few lines above:

```			const _position = object.position;

			return function update() {

				const position = scope.object.getWorldPosition( _position );
```

Reply:
Which means we can actually save this copy completely and just do:

```
				if(scope.object.parent)
					scope.object.parent.worldToLocal( position );
```

Reply:
> position is actually the object.position Vector3 i think because of this few lines above:

I'm afraid that's not true. `getWorldPosition()` returns a reference to the passed target vector.

Reply:
In any event, I see now why `scope.object.parent` has to be used when calling `worldToLocal()` 👍 .

Reply:
>> I'm afraid that's not true. getWorldPosition() returns a reference to the passed target vector.

Yes and the target vector is actually ``object.position``  because this is what we saved in ``const _position = object.position``. I probably misunderstood you [here](https://github.com/mrdoob/three.js/pull/23526#discussion_r843668853)

Reply:
> because this is what we saved in const _position = object.position in the outer scope of update().

Oh, I've missed that line! I think it would be less confusing to not save the position reference in the closure scope. It's a bit confusing since we normally don't do that.

Reply:
Great, removed it and tested with ``camera without parent``, ``camera in scene root``, ``camera with parent offset``